pkg/apis/batch/validation/validation_test.go

```diff
@@ -77,23 +77,73 @@ func getValidPodTemplateSpecForGenerated(selector *metav1.LabelSelector) api.Pod
 			Labels: selector.MatchLabels,
 		},
 		Spec: api.PodSpec{
-			RestartPolicy: api.RestartPolicyOnFailure,
-			DNSPolicy:     api.DNSClusterFirst,
-			Containers:    []api.Container{{Name: "abc", Image: "image", ImagePullPolicy: "IfNotPresent", TerminationMessagePolicy: api.TerminationMessageReadFile}},
+			RestartPolicy:  api.RestartPolicyNever,
```

Comment:
I think it's better to revert this and where needed you just replace the RestartPolicy.

Reply:
Ok, updated